### PR TITLE
Searchbar text

### PR DESCRIPTION
**Pull request by PilgrimMemoirs**
## Description

Resized the search bar text for small and medium search bars from 1.7rem to 1.5rem

## Additional information

* In search-bar.md HTML file: Added new class to medium search bar form tag: usa-search-medium
* In _search.scss Stylesheet: Added text-size properties on .usa-search-medium and .usa-search-small inputs. Set properties to 1.5rem. 
* Final product:

screen shot 2016-06-17 at 3 04 59 pm

* Orignally looked like this: 

searchbox_og

### CODE REVIEW

**Comment by PilgrimMemoirs** [comment]
Addresses issue #594

File: docs/_components/search-bar.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ lead: A block that allows users to search for specific content if they know what
 
   <div class="usa-grid">
     <div class="usa-width-one-half">
-      <form class="usa-search">
+      <form class="usa-search usa-search-medium">
```
Comment (maya) [inline docs/_components/search-bar.md]: The medium size is the default search size, so we do not want an additional class for it. Can you remove this and style `usa-search` directly?

**Comment by ericadeahl** [comment]
Text sizes look good to me! Thanks!

File: src/stylesheets/components/_search.scss [hunk 2]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
...
+    @include media($small-screen) {
+
+      [type="search"],
+      .usa-search-input {
+        font-size: 1.5rem;
+      }
+    }
+  }
+
+
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: Remove extra space.

File: src/stylesheets/components/_search.scss [hunk 3]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
...
       $width: $usa-btn-small-width;
 
       [type="search"],
       .usa-search-input {
         width: calc(100% - #{$width});
+        font-size: 1.5rem;
       }
 
+
+
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: Remove extra space.

**Comment by rogeruiz** [comment]
Thanks for the contribution @PilgrimMemoirs! There is one issue in the `src/stylesheets/components/_search.scss` file you updated.

```shell
$ npm test
# ... output from other tests
[11:19:18] 1 issues found in /Users/rogerruiz/Developer/federal-front-door/web-design-standards/src/stylesheets/components/_search.scss
[11:19:18] components/_search.scss:79 [W] Indentation: Line should be indented 2 spaces, but was indented 0 spaces
# ... output from other tests
```

@maya has some line notes around removing some extraneous lines as well. Once these are addressed, we can merge it all in 💯

File: src/stylesheets/components/_search.scss [hunk 4]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
     }
   }
 
+&.usa-search-medium {
+    @include media($small-screen) {
+
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: Remove this line break.

**Comment by maya** [comment]
@PilgrimMemoirs hello! Just following up, will you be able to make the changes above?

File: src/stylesheets/components/_search.scss [hunk 5]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
...
+      [type="search"],
+      .usa-search-input {
+        font-size: 1.5rem;
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: This should use the font size variable.

File: src/stylesheets/components/_search.scss [hunk 6]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
     }
   }
 
+&.usa-search-medium {
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: Should not use new class `.usa-search-medium`, should use default search class.

File: src/stylesheets/components/_search.scss [hunk 7]
```diff
@@ -76,15 +76,29 @@ $usa-btn-big-width:     11.6rem;
...
+
+
   &.usa-search-small {
     @include media($small-screen) {
       $width: $usa-btn-small-width;
 
       [type="search"],
       .usa-search-input {
         width: calc(100% - #{$width});
+        font-size: 1.5rem;
```
Comment (maya) [inline src/stylesheets/components/_search.scss]: This should use the font size variable.

**Pull request merged**